eppo_core/src/events/queued_event.rs

```diff
@@ -0,0 +1,47 @@
+use crate::events::event::Event;
+
+#[derive(Debug, Clone, PartialEq)]
+pub enum QueuedEventStatus {
+    PENDING,
+    RETRY,
+    FAILED,
```

Comment:
Rust's convention is CamelCase enum variants:
```suggestion
    Pending,
    Retry,
    Failed,
```